media_manager/models.py

```diff
@@ -7,7 +8,7 @@ class Course(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
 
-    class Meta:
+    class Meta(object):
```

Comment:
Great question.

In Python 3, any class that one writes will either be a subclass of a superclass that one explicitly specifies, or a subclass of the `object` class. As such, a class definition like `class Meta:` implicitly automatically subclasses `object`.

In Python 2, writing `class Meta:` is an old-style way of writing a class. This "old-style" way does not subclass the `object` class. But subclassing the `object` class [can be beneficial](https://docs.python.org/release/2.5.2/ref/node33.html), and so Python 2 introduced a "new-style" way of writing a class, i.e. `class Meta(object):`.

Given the above explanations, to make code python 2/3 compatible, if one's class has no superclass to explicitly inherit from, it is better to explicitly declare the `object` class as the superclass. As such, if one runs one's python 2/3 compatible code on Python 3, `class Meta:` and `class Meta(object):` will be practically identical. The former definition will automatically implicitly subclass `object`. Running the same code on python 2 however is a different story. Namely, the former will be "old-style" and the latter will be "new-style". For consistency, since Python 3 will process the class in a new-style way, then we should have python 2 process it in a new-style way too. Hence the explicit `object` subclassing.

Ref: https://docs.python.org/release/2.5.2/ref/node33.html and  https://stackoverflow.com/questions/4015417/python-class-inherits-object

Reply:
Awesome, makes sense. Thanks @joshuagetega !